Source/WebCore/svg/graphics/filters/SVGFilter.cpp

```diff
@@ -33,8 +33,7 @@
 
 namespace WebCore {
 
-static constexpr unsigned maxTotalNumberFilterEffects = 100;
-static constexpr unsigned maxCountChildNodes = 200;
```

Comment:
Can you explain this part of the change?

Reply:
Set maxCountChildNodes back to 200 and removing maxTotalNumberFilterEffects was explained in the commit message.